src/mf/mf-hid.cpp

```diff
@@ -481,6 +481,9 @@ namespace librealsense
                         safe_release(pSensor);
                     }
                 }
+                // ERROR_NOT_FOUND is normal if no sensors are available
+                else if( res != HRESULT_FROM_WIN32( ERROR_NOT_FOUND ) )
```

Comment:
How did you get to changing this file?? :)
Why will we have a device with no sensors?

Reply:
Redundant (and many) errors on L515 and D415

Reply:
👍